App/.settings/org.eclipse.core.resources.prefs

```diff
@@ -0,0 +1,3 @@
+eclipse.preferences.version=1
```

Comment:
This (and the other things in .settings/) you can _definitely_ add to .gitignore. Especially when working with others - I've had issues where 2 people had slightly different IDE/editor settings and they kept messing each other up haha.

Side note - are you using eclipse? If not then I have no clue how these things got here...

Reply:
That's a bit weird, I'm not using any IDE just plain old cloud shell. Either way, I'll git ignore it to prevent those issues 🙂